### PR TITLE
Lunch menu

### PR DESCRIPTION
**Pull request by theethaj**

### CODE REVIEW

**Comment by KOngTNP** [comment]
Good job!, everything is ok.

**Comment by Chayapol-c** [comment]
Well done!!!

**Comment by kidstylex** [comment]
Everything is ok. You can merge your branch to master.

File: Menu.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
 | Item                                   | Price |
 |:---------------------------------------|------:|
-| nothing yet                            |  0.0  |
-
-## Lunch Menu
-
-Coming soon.
-
-etc.
+| Lobster Bisque                         |  250.00 Baht |
```
Comment (kidstylex) [inline Menu.md:7]: THB

Comment (theethaj) [reply Menu.md]: I solved it already.

**Pull request merged**